index.html

```diff
@@ -1,29 +1,69 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
     <title>Document</title>
-</head>
-<body>
-    <h1 class="heading">All Awesome Books</h1>
-    <div class="showBooks" id="showBooks"></div>
-    <hr class="hr-line">
-    <div class="container">
-        <h2 class="heading">Add A New Book</h2>
-        <form action="#" id="my-form">
+  </head>
+  <body>
+      <main class="mainContainer">
+        <header>
+        <nav>
+            <p>Awesome Books</p>
+            <ul class="link">
+            <li><a href="#" class="list-nav">List</a></li>
+            <li>|</li>
+            <li><a href="#" class="books-nav">Add new</a></li>
+            <li>|</li>
+            <li><a href="#" class="contact-nav">Contact</a></li>
+            </ul>
+        </nav>
+        </header>
+    
+        <section>
+        <p id="time"></p>
+        </section>
+        <section>
+        <div class="book-list">
+            <h1 class="heading">All Awesome Books</h1>
+            <div class="showBooks" id="showBooks"></div>
+        </div>
+        <div class="container add-books">
+            <h2 class="heading">Add A New Book</h2>
+            <form action="#" id="my-form">
             <label for="title">
-                <input type="text" id="title" placeholder="Title" required>
+                <input type="text" id="title" placeholder="Title" required />
             </label>
-            <br>
+            <br />
             <label for="author">
-                <input type="text" id="author" placeholder="Author" required>
+                <input type="text" id="author" placeholder="Author" required />
             </label>
             <button type="submit" id="addBtn">Add</button>
-        </form>
-    </div>
-<script src="file.js"></script>
-</body>
-</html>
\ No newline at end of file
+            </form>
+        </div>
```

Comment:
- [OPTIONAL] As requested by the previous code reviewer, which is also a requirement for this project and a best practice for a front end developer(`sticking to the design as much as possible`), please kindly implement alternating background color for the book list items and shown in the image below. 

![alternatingBackground](https://user-images.githubusercontent.com/65852621/153370452-bceb1541-47a8-442c-843b-72b2277a9a1e.png)